OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs

```diff
@@ -20,8 +20,12 @@ public class OuterWildsRandomSpeedrun : ModBehaviour
         private string _goalPointName;
         private IModButton _speedrunButton; 
         private DateTime _startTime;
+        private DateTime _endTime;
```

Comment:
Can we initialize this to DateTime.MinValue for clarity

Reply:
yes